languages/rust/exercises/concept/tuples/.meta/design.md

```diff
@@ -3,8 +3,8 @@
 After completing the exercise, the user should:
 
 - know how to create tuples
-- understand tuple access syntax `a_tuple.0`
-- know how to destructure tuples into new variables, e.g. `let (x, y) = a_point;`
+- understand tuple access syntax `a_volume_pair.0`
+- know how to destructure tuples into new variables, e.g. `let (unit, measurement) = a_volume_pair;`
```

Comment:
Sorry to be confusing, but the design document is generic so the previous version was actually correct :)